contribution/item/community-lore/TRASH_LOWER_AMMO_CACHE

```diff
@@ -0,0 +1,2 @@
+Mere scrapes of metals and armor pieces from the dwelling people.
```

Comment:
"scrapes" should be changed to "scraps," no?

Reply:
^